documentation/create-official-version.adoc

```diff
@@ -0,0 +1,17 @@
+
+If you consider the status of the curriculum worthy of a new release or release candidate, you have to create a `tag` with Git.
+
+The format for a release candidate tag is as follows: +
+`2021.1-RCX`, where x is either 1 or the current release candidate number increased by one. +
+This tag will create a GitHub _pre-release_. They are [available here](https://github.com/isaqb-org/curriculum-foundation/releases) and provide you with a zip archive that contains the curriculum as PDF and HTML. +
+**ATTENTION:** You have to also append the current RC version to the version stored in the `document.version` file. If you forget to do that, the RC information will NOT be part of the document's version.
```

Comment:
That's not true anymore 🙂  We got rid of that file, the information is now extracted directly from the Git tag. 😃  Less stuff to track manually.

Reply:
removed that part (for release candidate). What about official releases?  I assume the `document.version` content is still used for that cases, isn't it?

Reply:
No, it is not. ☺️ if you look closely at your repository, the file isn't there anymore.

Reply:
sorry, you're right...